### ISSUE TITLE
1.3.0 fail to load linters

### ISSUE THREAD

**Issue by egeland** [atom-community/linter#801]
I'm running Atom 1.0.3 on 64bit Linux.
```
$ uname -a
Linux ceres 3.13.0-37-generic #64-Ubuntu SMP Mon Sep 22 21:28:38 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
$ atom --version
1.0.3
```

I upgraded linter to the latest version (1.3.0) and it stopped working.

I can reproduce by
```
apm uninstall linter
apm install linter@1.2.4
```
Linter works, loads `linter-rubocop`, and happily lints my files.
Then
```
apm uninstall linter
apm install linter
```
This installs 1.3.0 and nothing gets linted...

Here's the packages I have at that point:
```
/home/egeland/.atom/packages (5)
├── atom-material-syntax@0.2.1
├── atom-material-ui@0.5.13
├── linter@1.3.0
├── linter-rubocop@0.2.7
└── minimap@4.12.2
```

**Comment by egeland** [comment]
I should mention that I also tried another linter - `linter-ruby` and that didn't work with 1.3.0 either..

**Comment by steelbrain** [comment]
Yep, we've removed the legacy API completely, if you still want it, do
```
apm install linter@1.2.4
```

**Comment by egeland** [comment]
So I should raise this against the failing linters, I guess..

Thanks for creating linter, by the way, makes life so much easier!

**Comment by steelbrain** [comment]
Yes, I believe that's the right place for this issue.
and Pleasure :bow:
